modules/jolt_physics/objects/jolt_body_3d.cpp

```diff
@@ -1404,6 +1404,22 @@ void JoltBody3D::set_angular_damp(float p_damp) {
 	_update_damp();
 }
 
+void JoltBody3D::set_linear_damp_mode(DampMode p_mode) {
+	if (p_mode == linear_damp_mode) {
+		return;
+	}
+	linear_damp_mode = p_mode;
+	_update_damp();
+}
+
+void JoltBody3D::set_angular_damp_mode(DampMode p_mode) {
+	if (p_mode == angular_damp_mode) {
+		return;
+	}
+	angular_damp_mode = p_mode;
+	_update_damp();
+}
```

Comment:
This is a bit nitpicky of me (and perhaps a losing battle in the long run) but for the sake of matching the rest of the file stylistically, some more newlines might be appropriate.

```suggestion
void JoltBody3D::set_linear_damp_mode(DampMode p_mode) {
	if (p_mode == linear_damp_mode) {
		return;
	}

	linear_damp_mode = p_mode;

	_update_damp();
}

void JoltBody3D::set_angular_damp_mode(DampMode p_mode) {
	if (p_mode == angular_damp_mode) {
		return;
	}

	angular_damp_mode = p_mode;

	_update_damp();
}
```